<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="nSpinBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximum">
     <number>25000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="mSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="getAnswerButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Answer!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
